eqcorrscan/core/match_filter.py

```diff
@@ -51,6 +52,31 @@
 from eqcorrscan.core.lag_calc import lag_calc
 
 
+def _spike_test(stream, percent=0.99, multiplier=1e6):
```

Comment:
It's not, this was for an issue from openCV correlations - I'm not sure if it is needed for the scipy internals - I need to check that and if it's not needed I should remove it - it's really slow and at least one user has complained!